cursorless-talon/src/actions/actions_makeshift.py

```diff
@@ -56,13 +57,13 @@ class MakeshiftAction:
 
 @dataclass
 class TalonOptions:
-    post_command_sleep_ms: int = None
+    post_command_sleep_ms: Optional[int] = None
     await_command: bool = True
 
 
 def get_parameters(action: MakeshiftAction):
     command = action.vscode_command_id
-    command_options = {
+    command_options: dict[str, Any] = {
```

Comment:
Can the values be `Callable`?

Reply:
Ah. I'll see if I can't turn this into a dataclass or something at some point then.

Reply:
Ah right yeah that sounds good.  You mean as part of #725 I assume